### ISSUE TITLE
Follow the ARIA design pattern for tabs

### ISSUE THREAD

**Issue by Veyfeyken** [digipolisantwerp/acpaas-ui_react#219]
## Problem

The tabs component is missing ARIA roles and attributes. It is not accessible with assistive technology such as screenreaders.

## Suggestion

Follow the ARIA design pattern for tabs.

## Articles and demo

* [ARIA design pattern for tabs](https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel)
* [Example of tabs](https://www.w3.org/TR/wai-aria-practices-1.2/examples/tabs/tabs-1/tabs.html)
